src/intl.store.js

```diff
@@ -45,5 +46,12 @@ export default new Store( {
 			return msg.format( data );
 		}
 		return key;
+	},
+	getFormattedNumber( data, options ) {
+		const { currentLocale } = this.getState();
+		if ( data ) {
```

Comment:
This check should be removed as it will currently prevent rendering `0` as a value.